app/src/main/res/drawable-v24/ic_launcher_background.xml

```diff
@@ -0,0 +1,44 @@
+<!--
+  ~ /*
+  ~  * Nextcloud Talk application
+  ~  *
+  ~  * @author Mario Danic
+  ~  * Copyright (C) 2017-2020 Mario Danic <mario@lovelyhq.com>
```

Comment:
😕

Reply:
Backported and it is in the original file, so... left it in...